front/src/routes/integration/all/tasmota/device-page/index.js

```diff
@@ -0,0 +1,24 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import actions from '../actions';
+import TasmotaPage from '../TasmotaPage';
+import DeviceTab from './DeviceTab';
+
+@connect('user,tasmotaDevices,housesWithRooms,getTasmotaStatus', actions)
+class TasmotaIntegration extends Component {
+  componentWillMount() {
+    this.props.getTasmotaDevices(100, 0);
```

Comment:
What happens if there is more than 100 Tasmota devices? 

I know it's my fault, it used to be mandatory to give a "take" parameter in a get device request, but it's no longer the case! If you want all devices, remove the take it'll give you the full list :) 

Sorry, it's really my fault here ^^

Reply:
See https://github.com/GladysAssistant/Gladys/blob/master/server/lib/device/device.get.js#L65

Reply:
Ok, we'll need to check other services.